homeassistant/components/light/__init__.py

```diff
@@ -123,7 +123,7 @@
 
 
 PROFILE_SCHEMA = vol.Schema(
-    vol.ExactSequence((str, cv.small_float, cv.small_float, cv.byte))
+    vol.ExactSequence((str, cv.small_float, cv.small_float, cv.byte, cv.positive_int))
```

Comment:
Please make it backwards compatible, or else it will be a very big breaking change.

Reply:
As in: make the 5th argument optional (with default of 0) rather than required?

Reply:
Yes

Reply:
Took me awhile to figure out a voluptuous schema that would accept either scenario and still fail on improper data types. But eventually got a union type to work.

Mind if I push that approach (without tests) for a pre-approval before I invest time in the tests?

Reply:
Yes

Reply:
I did not see the pushed change? 

It should be something like

```python
vol.Any(
vol.ExactSequence((str, cv.small_float, cv.small_float, cv.byte, cv.positive_int))
vol.ExactSequence((str, cv.small_float, cv.small_float, cv.byte)),
)
```

Reply:
> I did not see the pushed change?

Oh, ha. When you said "Yes" earlier I thought it was "Yes, I would mind if you pushed without tests". Which I thought was a fair response :P But been having a very hard time getting vscode to run a single test in debug mode.

> It should be something like... vol.Any(...)

That's exactly what I landed on. Phew.

I made progress with vscode with `pip install -r requirements_test_all.txt`. But it takes ages to install, and I lose that environment when docker closes. Any advice there?

Reply:
I'll go ahead and push my current changes as-is and work on testing tonight.

Reply:
Changes pushed. (Note: the changes to `test_init.py` aren't done)

Reply:
Haha oops, sorry about the miscommunication.